pom.xml

```diff
@@ -27,7 +27,7 @@
         <url/>
     </scm>
     <properties>
-        <java.version>21</java.version>
+        <java.version>20</java.version>
```

Comment:
Wersja javy 20 nie jest LTS czyli long term support oznacza to ze nie musi ona byc dlugo wspierana, bez sensu jest uzywac wiec takich wersji w aplikacjach komercyjnych poniewaz szybko mozemy sie zlapac na tym ze jest jakis bug a okaze sie ze oni juz nie wspieraja tej wersji i i tak bedziemy musieli przeprowadzic migracje na wyzsza wersje javy